### PR TITLE
toolbox*: add Tamil translation

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): `0.0.99.3`**

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
Translations syntactically are correct. But toolbox original page requires some changes:

- `--version`, `--help` samples should be placed after all code examples
- `--version` should be after `--help`

These options are common so it's better to pay attention to more important command flags than mentioned above.

**Comment by EmilySeville7cfg** [comment]
Is there anybody who knows Tamil as @kbdharun?

File: pages.ta/linux/toolbox-list.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# toolbox list
+
+> ஏற்கனவே உள்ள `toolbox' கொள்கலன்களையும் படங்களையும் பட்டியலிடுங்கள்.
+> மேலும் விவரத்திற்கு: <https://manned.org/toolbox-list.1>.
+
+- அனைத்து `toolbox' கொள்கலன்களையும் படங்களையும் பட்டியலிடுங்கள்:
+
+`toolbox list`
+
+- `toolbox' கொள்கலன்களை மட்டும் பட்டியலிடுங்கள்:
```
Comment (navarroaxel) [inline pages.ta/linux/toolbox-list.md:10]:
```suggestion
- `toolbox` கொள்கலன்களை மட்டும் பட்டியலிடுங்கள்:
```

File: pages.ta/linux/toolbox-list.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# toolbox list
+
+> ஏற்கனவே உள்ள `toolbox' கொள்கலன்களையும் படங்களையும் பட்டியலிடுங்கள்.
+> மேலும் விவரத்திற்கு: <https://manned.org/toolbox-list.1>.
+
+- அனைத்து `toolbox' கொள்கலன்களையும் படங்களையும் பட்டியலிடுங்கள்:
```
Comment (navarroaxel) [inline pages.ta/linux/toolbox-list.md:6]:
```suggestion
- அனைத்து `toolbox` கொள்கலன்களையும் படங்களையும் பட்டியலிடுங்கள்:
```

Comment (navarroaxel) [reply pages.ta/linux/toolbox-list.md:6]: I'll merge these because are small typos.

**Review by navarroaxel — APPROVED** [review]

**Comment by kbdharun** [comment]
> Translations syntactically are correct. But [toolbox](https://github.com/tldr-pages/tldr/blob/main/pages/linux/toolbox.md) original page requires some changes:
> 
> * `--version`, `--help` samples should be placed after all code examples
> * `--version` should be after `--help`
> 
> These options are common so it's better to pay attention to more important command flags than mentioned above.

I will update it too, give me 10 minutes. 

> Is there anybody who knows Tamil as @kbdharun?

I think there are 2 people who had translated pages to Tamil before. They are active on GitHub but aren't active in the repo currently.  [Person 1](https://github.com/tldr-pages/tldr/commits?author=seenukarthi), [Person 2](https://github.com/tldr-pages/tldr/commits?author=arunisaac)

**Pull request merged**